corehq/apps/style/static/style/less/bootstrap2/includes/variables.less

```diff
@@ -81,11 +81,13 @@
 
 // Z-index master list
 // -------------------------
-// Used for a bird's eye view of components dependent on the z-axis
-// Try to avoid customizing these :)
-@zindexBrand:           900; // hq
-@zindexBrandHover:      950; // hq
-@zindexUIDatePicker:    1050; // display over modals
+// Also see Bootstrap 2's own master list:
+// corehq/apps/style/static/style/lib/bootstrap/less/variables.less
+@zindexDashboardSpinner:              500;
```

Comment:
not needed, since dashboard is B3, but I erred on the side of being conservative